### DISCUSSION TITLE
usb_claim_interface error -6

### DISCUSSION THREAD

**Discussion by Vendo232** [hertzg/rtl_433_docker#34] [Q&A] [answered]
Hi

I`m trying to setup RPI 4 ( or Zero W ) with Realtek RTL2838 DVB-T dongle and getting USB claim conflick.

Device:
![image](https://user-images.githubusercontent.com/44098230/135506991-63eee2b1-c88d-4d85-b5b3-7367996530f5.png)

`pi@raspberrypi:~ $ sudo docker run -it --rm --device /dev/bus/usb/001/003 hertzg/rtl_433:debian-buster-latest
Unable to find image 'hertzg/rtl_433:debian-buster-latest' locally
debian-buster-latest: Pulling from hertzg/rtl_433
4c2a0a79594a: Pull complete
d0917e1c3515: Pull complete
34797a5bd9e2: Pull complete
Digest: sha256:44a89ffc58ffcd36383487a7301281159d0d4f0d7def4054e172ba18366d6ee7
Status: Downloaded newer image for hertzg/rtl_433:debian-buster-latest
rtl_433 version 20.11 branch  at 202011122224 inputs file rtl_tcp RTL-SDR
Use -h for usage help and see https://triq.org/ for documentation.
Trying conf file at "rtl_433.conf"...
Trying conf file at "/root/.config/rtl_433/rtl_433.conf"...
Trying conf file at "/usr/local/etc/rtl_433/rtl_433.conf"...
Trying conf file at "/etc/rtl_433/rtl_433.conf"...
Registered 145 out of 175 device decoding protocols [ 1-4 8 11-12 15-17 19-21 23 25-26 29-36 38-60 63 67-71 73-100 10               2-105 108-116 119 121 124-128 130-149 151-161 163-168 170-175 ]
usb_claim_interface error -6
`
![image](https://user-images.githubusercontent.com/44098230/135506747-572fffb1-d98c-472b-b29b-c2164c292000.png)

**Comment by hertzg** [comment, marked as answer]
The device is probably already in use.


Reference: https://github.com/merbanan/rtl_433/issues/795
> The error -6 is LIBUSB_ERROR_BUSY. The device is already busy.
